<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="PT165LT" failures="1" tests="1" name="Test" time="9.784" errors="0" timestamp="2024-09-02T13:03:14 IST">
  <testcase classname="rahulshetty.TestLogin" name="Addtocartfuncationlity" time="3.034">
    <failure type="java.lang.AssertionError" message="The thank you message is incorrect expected [Thank you for your orde!] but found [Thank you for your order!]">
      <![CDATA[java.lang.AssertionError: The thank you message is incorrect expected [Thank you for your orde!] but found [Thank you for your order!]
at rahulshetty.TestLogin.Addtocartfuncationlity(TestLogin.java:93)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- Addtocartfuncationlity -->
</testsuite> <!-- Test -->
